src/array/_deprecated/reverse/reverse.ts

```diff
@@ -30,6 +30,7 @@ import type { Traversable } from '../../type/index.js'
  * @typeParam T - The element type.
  *
  * @group Array
+ * @deprecated Use `xs.reverse()` instead.
```

Comment:
what about iterators?

Reply:
`xs.toArray().reverse()`, that's what would have happened anyway